pyramid_oereb/lib/renderer/extract/json_.py

```diff
@@ -136,10 +136,15 @@ def _render(self, extract, param):
         if isinstance(extract.glossaries, list) and len(extract.glossaries) > 0:
             glossaries = list()
             for gls in extract.glossaries:
-                glossaries.append({
-                    'Title': self.get_multilingual_text(gls.title),
-                    'Content': self.get_multilingual_text(gls.content)
-                })
+                gls_title = self.get_multilingual_text(gls.title)[0]['Text']
+                if gls_title is not None:
+                    glossaries.append({
+                        'Title': self.get_multilingual_text(gls.title),
```

Comment:
Avid 2 call to `self.get_multilingual_text(gls.title)` ...

Reply:
OK, changed